core/schema-installer/src/main/java/org/hawkular/metrics/schema/Installer.java

```diff
@@ -76,34 +78,41 @@ public Installer() {
         replicationFactor = Integer.getInteger("hawkular.metrics.cassandra.replication-factor", 1);
         versionUpdateDelay = Long.getLong("hawkular.metrics.version-update.delay", 5) * 1000;
         versionUpdateMaxRetries = Integer.getInteger("hawkular.metrics.version-update.max-retries", 10);
+        waitRetry =  Long.getLong("hawkular.metrics.version-update.retry-delay", 10);
     }
 
     public void run() {
+        while (true) {
```

Comment:
I don't quite like how this could end up running forever without stopping, shouldn't we make use of `versionUpdateMaxRetries` for this cycle as well somehow (or some new variable)?

Reply:
Ohh a silly error, TBH I don't like either, I'll put a counter here.